packages/ui/src/components/SSelect/SSelectButton.vue

```diff
@@ -0,0 +1,121 @@
+<script setup lang="ts">
+import { computed } from 'vue-demi'
+import { useSSelectApi } from './api'
+import { SSelectButtonType } from './types'
+import SSelectChevron from './SSelectChevron'
+
+const props = withDefaults(
+  defineProps<{
+    type?: SSelectButtonType
+  }>(),
+  {
+    type: SSelectButtonType.Default,
+  },
+)
+
+const api = useSSelectApi()
+
+// const label = computed(() => api.label)
```

Comment:
Could you add TODO or remove this lines please?

Reply:
Sure.